packages/connector-alipay-native/src/constant.ts

```diff
@@ -33,8 +33,8 @@ export const defaultMetadata: ConnectorMetadata = {
   logo: './logo.svg',
   logoDark: null,
   description: {
-    en: 'Sign In with Alipay',
-    'zh-CN': '支付宝登录',
+    en: 'Allow your users to log in through their Alipay account.',
```

Comment:
should align 'log in' with 'sign in'? So do below?

Reply:
These descriptions will show when you choose a social connector to add. As a result, they should be short and slightly different from others, or users may find them dull.

Reply:
You can find similar descriptions when you try auth0 :P